lib/ansible/utils/module_docs_fragments/vexata.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright: (c) 2018, Sandeep Kasargod <sandeep@vexata.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    DOCUMENTATION = """
+options:
+    - See respective platform section for more details
+requirements:
+    - See respective platform section for more details
+notes:
+    - Ansible modules are available for Vexata VX100 arrays.
+"""
+
+    # Documentation fragment for Vexata VX100 series
+    VX100 = '''
+options:
+  array:
+    description:
+      - Vexata VX100 array hostname or IPv4 Address.
```

Comment:
Please add `type:` for all options
```suggestion
      - Vexata VX100 array hostname or IPv4 Address.
    type: str
```

Reply:
Sure will do